ha/util/conf_store.py

```diff
@@ -29,6 +30,17 @@
 from cortx.utils.log import Log
 from ha.core.error import ClusterCardinalityError
 
+
+# Confstore keys
```

Comment:
Some keys are already defined in k8s_setup/const.py 
Can you please have all keys in the same place

Reply:
Done. Moved Confstore class to k8s_setup/const.py